plaso/storage/file_interface.py

```diff
@@ -994,6 +901,9 @@ def RemoveProcessedTaskStorage(self, task):
     if self._storage_type != definitions.STORAGE_TYPE_SESSION:
       raise IOError('Unsupported storage type.')
 
+    if task.storage_format != definitions.STORAGE_FORMAT_SQLITE:
+      raise IOError('Unsupported storage format.')
```

Comment:
add "Unsupported storage format" as a reason to raise to the docstring

Reply:
Done